### PR TITLE
Create new mapping

### PR DESCRIPTION
**Pull request by MariaDolotova**

### CODE REVIEW

File: UI/src/app/services/common-utils.service.ts [hunk 1]
```diff
@@ -44,4 +49,27 @@ export class CommonUtilsService {
...
   }
+
+  openResetWarningDialog() {
+    const matDialog = this.matDialog.open(ResetWarningComponent, {
+      closeOnNavigation: false,
+      disableClose: false,
+      panelClass: 'warning-dialog',
+    });
+
+    matDialog.afterClosed().pipe(
```
Comment (etokareva) [inline UI/src/app/services/common-utils.service.ts:60]: I think you have no need in pipe and map in this case. Just do the same in subscribe

Comment (MariaDolotova) [reply UI/src/app/services/common-utils.service.ts]: Done!

File: UI/src/app/services/common-utils.service.ts [hunk 2]
```diff
@@ -44,4 +49,25 @@ export class CommonUtilsService {
...
+
+  openResetWarningDialog() {
+    const matDialog = this.matDialog.open(ResetWarningComponent, {
+      closeOnNavigation: false,
+      disableClose: false,
+      panelClass: 'warning-dialog',
+    });
+
+    matDialog.afterClosed().subscribe(res => {
+      if (res) {
```
Comment (etokareva) [inline UI/src/app/services/common-utils.service.ts:61]: We can use type coercing there and early return:
```ts 
if (!res) {
    return;
}
this.openSaveMappingDialog('save');
this.bridgeService.resetAllMappings();
this.storeService.resetAllData();
```

File: UI/src/app/components/popups/reset-warning/reset-warning.component.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+:host {
+    display: block;
+  }
+  
+  .reset-label {
```
Comment (etokareva) [inline UI/src/app/components/popups/reset-warning/reset-warning.component.scss:5]: please format this file

**Pull request merged**